### ISSUE TITLE
Feature request: underbar

### ISSUE THREAD

**Issue by berquist** [JP-Ellis/simpler-wick#4]
`simplewick` allows for placing contractions on the bottom

``` latex
\[
\bcontraction{a = }{b}{c d}{e}
a = b c d e
\]
```

to give

![screen shot 2016-10-03 at 2 02 28 pm](https://cloud.githubusercontent.com/assets/727571/19047945/14d7954a-8972-11e6-9e31-bd2a409ec72f.png)

If this is easy for you to implement, it would be a great addition.

**Comment by JP-Ellis** [comment]
Sorry for not getting back to you earlier.

I will make sure to add this feature in the next version, it should be easy enough to do.

**Comment by latrix** [comment]
Hi Joshua, 

Simpler-wick is awesome. My latex codes for representing Wick contraction is simplified so much. However, I still have a question. Is it possible to set an optional variable to adjust the width of contraction line?  

I can give you the comparison between the simple wick and simpler wick:

<img width="575" alt="wick" src="https://cloud.githubusercontent.com/assets/16337006/24256178/d14da74c-0fbd-11e7-8596-62f7f9099b38.png">

**Comment by melsophos** [comment]
I agree that this would be an excellent addition, especially as an alternative to the problem described in #11.

**Comment by evanberkowitz** [comment]
I often want to write "two contractions at once"---one contraction above, and another contraction below.
